Made changes to grid2d class

kaustabpal opened the pull request:
- If target is inside an obstacle it is shifted to the nearest free cell

kaustabpal committed:
Made changes to grid2d class

orensbruli commented:
@kaustabpal 
If you check your commit you will see that several *.so, *.in, and other binary and generated files have been uploaded to the repo in this commit. It' 's not what is expected. Only source code and some config files should be uploaded to the repo. Can you please fix your commit?
Best.

kaustabpal commented:
Hi,
I have cleaned up the binary and generated files and added their extensions to the gitignore file. Here is the [pull request](https://github.com/robocomp/robocomp/pull/379).
